docs/automated-process.md

```diff
@@ -11,6 +11,34 @@ release process. The automated processes uses GitHub Actions to automate the
     `${PLACEHOLDER_VALUE}` where `PLACEHOLDER_VALUE` describes the value to
     specify.
 
+## Setup for the release
+
+### Credentials
+
+- Access to https://github.com/ansible-community/ansible-build-data repo
```

Comment:
Specifically, they need to be a member of https://github.com/orgs/ansible-community/teams/release-management-wg/ (@ansible-community/release-management-wg). That gives the necessary permissions to this repo